OmiseExt.php

```diff
@@ -20,7 +20,7 @@ class OmiseExt extends SC_Plugin_Base
      * @param  array $arrPlugin plugin_infoを元にDBに登録されたプラグイン情報(dtb_plugin)
      * @return void
      */
-    public function install($arrPlugin)
+    public static function install($arrPlugin, $objPluginInstaller = null)
```

Comment:
Since we are changing these methods to `static`, are these required for ECube `2.12.5` and `2.13.3` as well or just `2.17`?

Reply:
@aashishgurung 

Just for 2.17.

Both ECCube 2.12.5 and 2.13 require PHP 5.X.
These version are already out of support from PHP official.
So, this fix is for EC-CUBE ver 2.17.
It works with current supported PHP version.

Reply:
@shunsuke-h-opn @aashishgurung what will happen for current users of PHP 5.x who migrate to this version?

Reply:
@shunsuke-h-opn 
I will ask it to EC-CUBE official support team.

Reply:
`static` method was available since PHP 5.x. So, this should work if 2.17 version of EC-CUBE allows PHP 5.x.

Reply:
My only concern is that the way `install` method needs to be implemented after this change. Before, we would do like following

```
$obj = new OmiseExt;

$obj->install(<params>)
```

But, with this change, we have to do like

```
OmiseExt::install(<params>)
```

So, if we release with this change and a merchant with legacy EC-CUBE version, i.e a version before 2.17, upgrades our plugin then this might break his/her implementation. We do have a few merchants using EC-CUBE older versions.